### DISCUSSION TITLE
Glossary for individual languages for more consistent translations

### DISCUSSION THREAD

**Discussion by earomc** [anyproto/contributors#64] [Localization]
To make translations of terms like "Block", "Relation", "Object" etc. more consistent across different translations, there should be a glossary for each language that is easily visible for every translator. For example, I work on German. It could be confusing for the user if "Relation" is sometimes translated to "Beziehung" and sometimes to "Relation". I think you get the gist.

**Comment by Shampra** [comment]
Hi, 
Look here, same problem 😉
https://github.com/orgs/anyproto/discussions/45#discussioncomment-6667742
